lib/amazon_mws.rb

```diff
@@ -12,7 +12,7 @@
 
 class AmazonMwsOrderResponse
   def initialize(response)
-    @response = response.fetch("GetOrderReferenceDetailsResponse", {})
+    @response = Hash.from_xml(response.body).fetch("GetOrderReferenceDetailsResponse", {})
```

Comment:
Shouldn't this be wrapped in the above class? Then you can use directly the `#body`, which will be parsed